### DISCUSSION TITLE
Akku-Ladezustand in kWh

### DISCUSSION THREAD

**Discussion by norbertle** [solectrus/solectrus#2579] [Ideas]
Ich bin mir nicht sicher ob das so gewünscht ist oder vielleicht ein "Fehler" ist.
Wenn ich mir in der Ansicht TAG Akku in kW ansehe geht das von -1 bis +1  (oder -x bis +x) 
 Für mich ist das der Akkuverlauf wo Aufladung und Entladung als die Veränderung dargestellt wird.
 
 Hilfreich wäre vielleicht eine Anzeige Akku in kWh so das man den Verlauf der im Speicher vorhanden Energie hätte. Damit könnte man sehen ob der Akku wirklich komplett voll ist bzw. z. B. die 70% Grenze erreicht hat.

**Comment by ledermann** [comment]
Das Diagramm "Akku in KW" zeigt die Ladung und Beladung des Akkus im Zeitverlauf. Positive Werte (oberhalb der Null-Linie) zeigen eine Beladung an, negative Werte (unterhalb) eine Entladung. Die Summe (in kWh) steht links in der Strombilanz:

![CleanShot 2023-11-30 at 07 50 41@2x](https://github.com/solectrus/solectrus/assets/3557/fd941725-0311-4952-9a6a-8edf4857efb9)

Für den Ladestand des Akkus im Zeitverlauf gibt es ein separates Diagramm, "Ladestand in %":

![CleanShot 2023-11-30 at 07 54 54@2x](https://github.com/solectrus/solectrus/assets/3557/b639297c-c0a1-43a9-9edb-bd632eda71aa)

Ich denke, das sollte genau das sein, was du suchst.

**Reply by norbertle** [reply]
Das mit den Darstellung ist klar und verstanden. Mir geht es wirklich um die Beladung des Speichers in KWh und nicht in %.
Bei den % muss man ja dann rechnen. Kapazität x Konditionierung Faktor (70%) und dann mal % des Ladezustandes. 
Da ist der Wert der kWh im Speicher doch hilfreich. Auf jeden Fall für mich.

**Reply by ledermann** [reply]
Ok, jetzt verstehe ich. Allerdings ist es so, dass der Ladezustand vom SENEC nur als Prozentwert geliefert wird, nicht in kWh. Um die kWh anzuzeigen, müsste SOLECTRUS das selbst berechnen, was aber die Akkukapazität als Ausgangsgröße erforderlich macht. Diese müsste also irgendwo erfasst werden, derzeitig gibt es diesen Wert gar nicht.

Könnte man machen, aber mich überzeugt das nicht so wirklich. Bei anderen Geräten mit Akku (Laptop, Smartphone, E-Auto) sieht man auch nur den Ladestand in Prozent. Daran ist man gewöhnt. Also ich jedenfalls ;-)

Aber mal sehen, ob noch Feedback von anderen Nutzen kommt.

**Reply by StevieC121176** [reply]
Ich muss gestehen ich finde die Anzeige der Kapazität auch besser. Vielleicht könnte man das in so einem Tooltip anzeigen. 
Um die einmalige Eingabe der Größe wird man allerdings nicht herumkommen. Ich habe eine Kostal/Byd Kombination und auch hier kann man die Größe nicht direkt auslesen.

**Reply by ledermann** [reply]
Ok, den Ladestand in kWh als Tooltip des Ladestand-Kreises anzuzeigen, finde ich eine gute Idee, das wäre auch leicht zu machen. Die Kapazität müsste dann noch irgendwo eingegeben werden können. Ich habe #2608 angelegt.

**Reply by norbertle** [reply]
Danke.
